app/javascript/packs/admin.js

```diff
@@ -1,5 +1,10 @@
 import { Application } from 'stimulus';
 import { definitionsFromContext } from 'stimulus/webpack-helpers';
+import Rails from '@rails/ujs';
+
+// Initialize Rails unobtrusive scripting adapter
+// https://github.com/rails/rails/blob/main/actionview/app/assets/javascripts/README.md#es2015
+Rails.start();
```

Comment:
where the magic happens

Reply:
![image](https://user-images.githubusercontent.com/47985/120297298-2b0c4d00-c2f3-11eb-88c9-13083679a2c7.png)